src/actions/map-style-actions.ts

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2022 Uber Technologies, Inc.
```

Comment:
This file is not recognized as a move. Probably too many changes. You may want to first do a PR that renames it (even if it breaks CI) and then land this?

Reply:
Please address this